.github/workflows/ci.yml

```diff
@@ -47,11 +47,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: dtolnay/rust-toolchain@nightly
+      - uses: dtolnay/rust-toolchain@stable
       - name: Check documentation
         env:
           RUSTDOCFLAGS: -D warnings
-        run: cargo +nightly doc --no-deps --document-private-items
+        run: cargo doc --no-deps --document-private-items -- -D warnings
```

Comment:
It seems the `-D warnings` is not recognized by `cargo doc`. What is it for? Can we remove it?

Reply:
Thanks, i forgot to remove that. With only the rustdoc flags:

```console
home ~/projects/pubgrub ⊥  cargo doc --no-deps --document-private-items
 Documenting pubgrub v0.2.1 (/home/konsti/projects/pubgrub)
warning: unresolved link to `NotThere`
   --> src/lib.rs:218:7
    |
218 | //! [`NotThere`]
    |       ^^^^^^^^ no item named `NotThere` in scope
    |
    = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`
    = note: `#[warn(rustdoc::broken_intra_doc_links)]` on by default

warning: `pubgrub` (lib doc) generated 1 warning
    Finished dev [unoptimized + debuginfo] target(s) in 0.19s
   Generated /home/konsti/projects/pubgrub/target/doc/pubgrub/index.html
home ~/projects/pubgrub ⊤  RUSTDOCFLAGS="-D warnings" cargo doc --no-deps --document-private-items
 Documenting pubgrub v0.2.1 (/home/konsti/projects/pubgrub)
error: unresolved link to `NotThere`
   --> src/lib.rs:218:7
    |
218 | //! [`NotThere`]
    |       ^^^^^^^^ no item named `NotThere` in scope
    |
    = help: to escape `[` and `]` characters, add '\' before them like `\[` or `\]`
    = note: `-D rustdoc::broken-intra-doc-links` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(rustdoc::broken_intra_doc_links)]`

error: could not document `pubgrub`

Caused by:
  process didn't exit successfully: `/home/konsti/.rustup/toolchains/stable-x86_64-unknown-linux-gnu/bin/rustdoc --edition=2021 --crate-type lib --crate-name pubgrub src/lib.rs -o /home/konsti/projects/pubgrub/target/doc --error-format=json --json=diagnostic-rendered-ansi,artifacts,future-incompat --diagnostic-width=136 --document-private-items -C metadata=864e88f22e1ea1c7 -L dependency=/home/konsti/projects/pubgrub/target/debug/deps --extern indexmap=/home/konsti/projects/pubgrub/target/debug/deps/libindexmap-7b47b1eefefde8a2.rmeta --extern log=/home/konsti/projects/pubgrub/target/debug/deps/liblog-1a8c0ad5b258c07f.rmeta --extern priority_queue=/home/konsti/projects/pubgrub/target/debug/deps/libpriority_queue-1a2f8edad3532bd2.rmeta --extern rustc_hash=/home/konsti/projects/pubgrub/target/debug/deps/librustc_hash-9ddd449969ae8a63.rmeta --extern thiserror=/home/konsti/projects/pubgrub/target/debug/deps/libthiserror-81fad4d2bc5b2426.rmeta -D warnings --crate-version 0.2.1` (exit status: 1)
```

Reply:
```suggestion
        run: cargo doc --no-deps --document-private-items
```